.github/workflows/deploy-all.yml

```diff
@@ -8,6 +8,7 @@ on:
         required: true
         default: ""
       environment:
+        description: 'Openshift Environment'
```

Comment:
If `Openshift Environment` is related to the `Openshift Namespace` it would not be accurate because we have DEV/TEST/PROD namespaces but we have, for instance, DEV/TEST/QA environments.
I would avoid calling this `Openshift Environment`.